README.md

```diff
@@ -197,6 +198,20 @@ FLAGS:
 example: heimdall_tools jfrog_xray_mapper -j xray_results.json -o xray_results_hdf.json
 ```
 
+## aws_config_mapper
```

Comment:
Please include details from your excellent documentation from `aws_config_mapper.md` here in the Readme as well.

Reply:
done